### PR TITLE
Add boolean option to allow multiple lookups

### PR DESCRIPTION
**Pull request by petechd**
### PR Context
This adds new boolean option to question block that allows multiple lookups selection in textfield autosuggest component. It is required by [runner PR](https://github.com/ONSdigital/eq-questionnaire-runner/pull/487). This change is described on [Trello card](https://trello.com/c/RqJlAYzx). New invalid test schema was added.

### Checklist

* [ ] eq-translations updated to support any new schema keys which need translation

### CODE REVIEW

**Comment by petechd** [comment]
[FEATURE: Socio Demographic look-ups - Allow multiple selections (S)](https://trello.com/c/RqJlAYzx/4407-feature-socio-demographic-look-ups-allow-multiple-selections-s)

File: schemas/answers/text_field.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
       "suggestions_url": {
         "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
       },
+      "allow_multiple": {
+        "type": "boolean"
+      },
```
Comment (MebinAbraham) [inline schemas/answers/text_field.json:27]: Could `allow_multiple` be conditional on the existence of `suggestions_url`?

**Review by MebinAbraham — APPROVED** [review]

File: schemas/answers/text_field.json (hunk 1 above)
Comment (ajmaddaford) [reply schemas/answers/text_field.json:27]: We should consider nesting `url` and `allow_multiple` under a `suggestions` key, then `url` can be mandatory if `suggestions` exist:

```json
"suggestions": {
  "url": "",
  "allow_multiple": ""
}
```

Comment (MebinAbraham) [reply schemas/answers/text_field.json:27]: It would make the schema less ambiguous, so I think it would be beneficial to update.

Comment (petechd) [reply schemas/answers/text_field.json]: I've added nested url in suggestions and dependency of suggestion on url, need to fix runner as schemas and resolve script are already fixed and seems like working with new validator.

File: schemas/answers/text_field.json [hunk 2]
```diff
@@ -19,8 +19,21 @@
...
+        "properties": {
+          "url": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+          },
+          "allow_multiple": {
+            "type": "boolean"
+          }
+        },
+        "dependencies": {
+          "suggestions": {
```
Comment (ajmaddaford) [inline schemas/answers/text_field.json:33]: I don't think dependencies is needed as `url` is nested inside `suggestions`. Just marking `url` as required should only make it mandatory if the optional `suggestions` property exists.

Comment (petechd) [reply schemas/answers/text_field.json]: I marked `url` as required now, dependency removed. Thanks!

**Review by ajmaddaford — APPROVED** [review]

File: tests/schemas/invalid/test_invalid_text_field_suggestions.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
+{
```
Comment (MebinAbraham) [inline tests/schemas/invalid/test_invalid_text_field_suggestions.json:1]: Is this schema needed?. We don't normally commit schemas to test raw JSON schema validation. As you are just testing the "required" rule I don't think we need this.

Comment (petechd) [reply tests/schemas/invalid/test_invalid_text_field_suggestions.json]: Schema removed, thanks!

**Review by MebinAbraham — APPROVED** [review]

**Comment by MebinAbraham** [comment]
Don't merge this until runner and schemas are ready.

**Pull request merged**